### PR TITLE
[Fix #215] Add versioning stragety rules

### LINKED ISSUE #215: Prepare gem to 1.0.0 version
- [x] update dependencies
- [x] update versioning rule
- [x] add changelog directory logs for 0-versions
- [x] check contributing file
- [x] review issues

### PR DESCRIPTION
**Pull request by roman-dubrovsky**
Before you submit a pull request, please make sure you have to follow:

- [x] read and know items from the [Contributing Guide](https://github.com/datarockets/datarockets-style/blob/master/CONTRIBUTING.md#pull-requests)
- [x] add a description of the problem you're trying to solve (short summary from related issue)
- [x] verified that cops are ordered by alphabet
- [x] add a note to the style guide docs (if it needs)
- [x] add a note to the changelog file
- [x] the commit message contains the number of the related issue (if it presents)
  and word `Fix` if this PR closes related issue
- [ ] squash all commits before submitting to review

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -15,3 +15,19 @@ gem push *built_file*
 * Announce the new release,
    making sure to say "thank you" to the contributors
    who helped shape this version!
+
+## Versioning stragety
+
+As versioning stragety, we're using SemVer: `MAJOR.MINOR.PATCH`
+
+A `MAJOR` vesrion must be incremented if there are any backward-incompatible breaking changes included in a release. This has the benefit of making it easy for anyone to quickly identify if a new version will work differently than a previous one.
```
Comment (texpert) [inline RELEASING.md:23]:
```suggestion
A `MAJOR` version must be incremented if there are any backward-incompatible breaking changes included in a release. This has the benefit of making it easy for anyone to quickly identify if a new version will work differently than a previous one.
```

File: RELEASING.md [hunk 2]
```diff
@@ -15,3 +15,19 @@ gem push *built_file*
...
+
+Usually, we'll incremented it on updating major versions of rubocop dependecies. Ideally, it should be synchronize with major verion of `rubocop`.
```
Comment (texpert) [inline RELEASING.md:25]:
```suggestion
Usually, we'll increment it on updating major versions of rubocop dependencies. Ideally, it should be synchronized with the major verion of `rubocop`.
```

File: RELEASING.md [hunk 3]
```diff
@@ -15,3 +15,19 @@ gem push *built_file*
...
+
+The `MINOR` version must be incremented if there are backward-compatible functionality is introduced. In the strictest sense, this means you should be able to upgrade to a new minor version without experiencing any breaking changes.
```
Comment (texpert) [inline RELEASING.md:27]:
```suggestion
The `MINOR` version must be incremented if backward-compatible functionality is introduced. In the strictest sense, this means you should be able to upgrade to a new minor version without experiencing any breaking changes.
```

File: RELEASING.md [hunk 4]
```diff
@@ -15,3 +15,19 @@ gem push *built_file*
...
+
+Usually, we'll incremented it on changing list of cops, updating minor versions of dependencie.
```
Comment (texpert) [inline RELEASING.md:29]:
```suggestion
Usually, we'll increment it on changing the list of cops, updating minor versions of dependencies.
```

File: RELEASING.md [hunk 5]
```diff
@@ -15,3 +15,19 @@ gem push *built_file*
...
+
+The `PATCH` version is meant for backward-compatible bug fixes. You should expect no new functionality with a new patch version, only improvements.
+
+Usually, we'll use it for updating pathc version of depencies or fixing bugs of the last version. Change rules of cops in most cases should be added to minor version.
```
Comment (texpert) [inline RELEASING.md:33]:
```suggestion
Usually, we'll use it for updating patch versions of dependencies or fixing bugs of the last version. Changed rules of cops in most cases should be added to minor version.
```

**Review by texpert — APPROVED** [review]

**Comment by roman-dubrovsky** [comment]
@texpert nice, thank you so much for your reviewing ❤️

File: RELEASING.md [hunk 6]
```diff
@@ -15,3 +15,19 @@ gem push *built_file*
 * Announce the new release,
    making sure to say "thank you" to the contributors
    who helped shape this version!
+
+## Versioning stragety
+
+As versioning stragety, we're using SemVer: `MAJOR.MINOR.PATCH`
```
Comment (texpert) [inline RELEASING.md:21]: Oops - another one:
```suggestion
As versioning strategy, we're using SemVer: `MAJOR.MINOR.PATCH`
```

**Review by lazycoder9 — APPROVED** [review]

**Pull request merged**